release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+  ingress:
+  - {}
+----
+
+See
+xref:../admin_guide/managing_networking.adoc#admin-guide-manage-networking[Managing
+Networking] for more information.
+
+[[ocp-27-cluster-ip-range-more-flexible]]
+==== Cluster IP Range Now More Flexible
+
+Cluster IP ranges are now more flexible by allowing multiple subnets for hosts.
+This provides the capability to allocate multiple, smaller IP address ranges for
+the cluster. This makes it easier to migrate from one allocated IP range to
+another.
+
+There are multiple comma-delimited CIDRs in the configuration file. Each node is
+allocated only a single subnet from within any of the available ranges. You can
+not allocate different-sized host subnets, or use this to change the host subnet
+size The `clusterNetworkCIDRs` can be different sizes, but must be equal to or
+larger than the host subnet size It is not allowed to have some nodes use
+subnets that are not part of the `clusterNetworkCIDRs`.
+
+In regard to migration or edits, networks can be added to the list, CIDRs in the
+list may be re-ordered, and a CIDR can be removed from the list when there are
+no nodes that have an SDN allocation from that CIDR.
+
+Example:
+
+----
+clusterNetworkCIDR: 10.1.0.0/24, 10.1.5.0/24, 10.2.1.0/24 …
+----
+
+[[ocp-37-routes-alloed-to-set-cookie-names-for-session-stickiness]]
+==== Routes Allowed to Set Cookie Names for Session Stickiness
+
+The HAProxy router can look for a cookie in a client request. Based on that
+cookie name and value, always route requests that have that cookie to the same
+pod instead of relying upon the client source IP, which can be obscured by an F5
+doing load balancing.
+
+A cookie with a unique name is used to handle session persistence.
+
+. Set a per-route configuration to set the cookie name used for the session.
+. Add an `env` to set a router-wide default.
+. Ensure that the cookie is set and honored by the router to control access.
+
+Example scenario:
+
+. Set a default cookie name for the HAProxy router:
++
+----
+$ oc env dc/router ROUTER_COOKIE_NAME=default-cookie
+----
+
+. Log in as a normal user and create the project/pod/svc/route:
++
+----
+$ oc login user1
+$ oc new-project project1
+$ oc create -f https://example.com/myhttpd.json
+$ oc create -f https://example.com/service_unsecure.json
+$ oc expose service service-unsecure
+----
+
+. Access the route:
++
+----
+$ curl $route -v
+----
++
+The HTTP response will contain the cookie name. For example:
++
+----
+Set-Cookie: default_cookie=[a-z0-9]+
+----
+
+. Modify the cookie name using route annotation:
++
+----
+$ oc annotate route service-unsecure router.openshift.io/cookie_name="route-cookie"
+----
+
+. Re-access the route:
++
+----
+$ curl $route -v
+----
++
+The HTTP response will contain the new cookie name:
++
+----
+Set-Cookie: route-cookie=[a-z0-9]+
+----
+
+See
+xref:../architecture/networking/routes.adoc#route-specific-annotations[Route-specific
+Annotations] for more information.
+
+[[ocp-37-hsts-policy-support]]
```

Comment:
Should this be it's own section in the docs (under router)? And the note is to point to it?